src/pages/enterprise/auth-connect/azure-ad.md

```diff
@@ -0,0 +1,116 @@
+
+<command-line>
+<command-prompt>ionic cordova plugin add @ionic-enterprise/auth --variable AUTH_URL_SCHEME=com.company.app</command-prompt>
+</command-line>
+
+### Configure Auth Connect
+
+It's recommended to create an `AuthenticationService` class that encapsulates Azure AD and Ionic Auth Connect’s login functionality.
+
+Generate this class using the `ionic generate` command:
+
+<command-line>
+<command-prompt>ionic generate service services/authentication</command-prompt>
+</command-line>
+
+Extend the `IonicAuth` class, then configure all Azure AD details in the `IonicAuthOptions` object:
+
+```typescript
+import { IonicAuth, IonicAuthOptions } from '@ionic-enterprise/auth';
+
+export class AuthenticationService extends IonicAuth {
+
+constructor() {
+    const azureConfig: IonicAuthOptions = {
+        // The auth provider.
+        authConfig: 'azure',
+        // The platform which the app is running on
+        platform: 'cordova',
+        // client or application id for provider
+        clientID: 'FILL_IN',
+        // the discovery url for the provider
+        // OpenID configuration
+        discoveryUrl: 'FILL_IN',
+        // the URI to redirect to after log in
+        redirectUri: 'FILL_IN',
+        // requested scopes from provider
+        scope: 'FILL_IN',
+        // the audience, if applicable
+        audience: 'FILL_IN',
+        // the URL to redirect to after log out
+        logoutUrl: 'FILL_IN',
+        // The type of iOS webview to use. 'shared' will use a webview that can 
+        // share session/cookies on iOS to provide SSO across multiple apps but
+        // will cause a prompt for the user which asks them to confirm they want
+        // to share site data with the app. 'private' uses a webview which will
+        // not prompt the user but will not be able to share session/cookie data
+        // either for true SSO across multiple apps.
+        iosWebView: 'private',
+        // required if running on the Web
+        clientSecret: ''
+    };
+    
+    super(auth0Config);
+    }
+}
+```
+
+Some of these `IonicAuthOptions` values are unique, and must be set based on your Azure AD app’s details:
+
+* `platform`: Use “cordova” or “capacitor” accordingly.
+* `clientID`: Your app’s _Application ID_. Example: cebbb0be-d578-4bbd-9712-4b0fe05c06aa
+* `redirectUri`: The URI to redirect to after the user has logged in. Use the same AUTH_URL_SCHEME variable value (App Id) from when the Auth Connect plugin was installed. Example: com.company.app://callback
+* `logoutUrl`: The URI to redirect to after the user has logged out. Example: com.company.app://login?logout=true
+* `audience`: Your custom API URL of choice, such as `https://api.myapp.com`.
+* `scope`: Unlock access to protected resources, such as read/write permissions. Example: openid offline_access email picture profile
```

Comment:
Scope: in the CS Demo app, we also have "https://vsquad.onmicrosoft.com/api/Hello.Read", which appears to be a custom Published Scope in our private Azure org. Is something like this required, or is the default assigned "user_impersonation" good enough?

Reply:
you need at least `offline_access` that one is required to use tokens.